sonar-csharp-plugin/src/main/java/org/sonar/plugins/csharp/CSharpSensor.java

```diff
@@ -88,7 +90,7 @@ private boolean shouldExecuteOnProject(FileSystem fs) {
 
   private void executeInternal(SensorContext context) {
     Optional<Path> roslynReportPath = config.roslynReportPath();
-    Optional<Path> protobufReportsDirectory = config.protobufReportPath();
+    Optional<Path> protobufReportsDirectory = config.protobufReportPath(false);
```

Comment:
Could you just let us know what this `false` means?

Reply:
ok, I'm refactoring it and creating two different methods